source/brailleDisplayDrivers/lilli.py

```diff
@@ -76,14 +76,14 @@ def _handleKeyPresses(self):
 			except:
 				pass
 			if not key: break
-			if (key <= 64) or ((key >= 257) and (key <= 296)): 
+			if (key <= 0x40) or ((key >= 0x101) and (key <= 0x128)): 

```

Comment:
You don't need parentheses around the comparisons.

Also, the second half could be written more compactly:
`0x101 <= key <= 0x128`
I find that a bit more readable. Also, this means you can remove all parentheses.

There is trailing whitespace here.